From b11560df1d7d9d166f962fa337bbdd355d91edfc Mon Sep 17 00:00:00 2001
From: Jason Liu <r64343@freescale.com>
Date: Wed, 11 Sep 2013 10:50:09 +0800
Subject: [PATCH 0942/1543] MLK-9750 ARM: Kconfig: increase
 FORCE_MAX_ZONEORDER for ARCH_MXC

commit 5e64348a8adee1359a92b984ce71cad07c8cba16 from
git://git.freescale.com/imx/linux-2.6-imx.git

Need increase the FORCE_MAX_ZONEORDER to 14 for high resolution camera
(GPU 2D user case). The default value 11(4MB) is not enough now.

Signed-off-by: Jason Liu <r64343@freescale.com>
(cherry picked from commit fff642ffe868cb55f5caec0501e36fd28b6ece50)
---
 arch/arm/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index 03a6a75..9e90b97 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -1841,6 +1841,7 @@ config FORCE_MAX_ZONEORDER
 	range 11 64 if ARCH_SHMOBILE_LEGACY
 	default "12" if SOC_AM33XX
 	default "9" if SA1111 || ARCH_EFM32
+	default "14" if ARCH_MXC
 	default "11"
 	help
 	  The kernel memory allocator divides physically contiguous memory
-- 
1.7.5.4

